.github/workflows/publish-olm-bundle.yaml

```diff
@@ -0,0 +1,80 @@
+name: Submit release OLM bundle
+
+on:
+  release:
+    types:
+      - published
+# Remove this trigger event once workflow behavior is validated
+  issues:
+    types:
+      - opened
+
+jobs:
+  createPullRequest:
+    name: Publish new OperatorHub release
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Determine Repository Variable
+      # Remove this conditional logic once the workflow behavior
+      # is validated
+          run: |
+            if [[ "${{ github.event_name }}" == "issues" && "${{ github.event.action }}" == "opened" ]]; then
+              echo "repository=fmuyassarov/community-operators" >> $GITHUB_ENV
+            else
+              echo "repository=k8s-operatorhub/community-operators" >> $GITHUB_ENV
+            fi
+
+      - name: Build the bundle
+        run: pushd deployment/operator && VERSION=0.8.1 make bundle && popd
+
+      - name: Checkout upstream community-operators repo
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ env.repository }}
+          path: community-operators
+          ref: main
+          token: ${{ secrets.BOT_PAT }}
+
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v6
+        id: import-gpg
+        with:
+          gpg_private_key: ${{ secrets.BOT_GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.BOT_GPG_PASSPHRASE }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+          workdir: community-operators
+
+      - name: Copy the bundle to the community-operators repo
+        run: |
+          mkdir -p community-operators/operators/nri-plugins-operator/v0.8.1
+          cp -r deployment/operator/bundle/ community-operators/operators/nri-plugins-operator/v0.8.1
+# mkdir -p community-operators/operators/nri-plugins-operator/${{ github.ref_name }}
+# cp -r deployment/operator/bundle/ community-operators/operators/nri-plugins-operator/${{ github.ref_name }}
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          signoff: true
+          author: nri-plugins-bot
```

Comment:
I think this will fail with `The format of 'nri-plugins-bot' is not a valid email address with display name.`

Reply:
I have a fix for it here: https://github.com/klihub/nri-plugins/commit/090e942eabdde4e5de67121f8122d0d421bc2795.

@fmuyassarov I also played around (maybe more like wasted a few hours) with some ideas of how we could keep the test-trigger around. It is here if you're interested: https://github.com/klihub/nri-plugins/commit/cff767c9e1c27439c4aed374d52e8b2acfaa3999

Reply:
Good catch about the author. Apparently I forgot to add an author in a format that git understans even though I was using it in my tests https://github.com/fmuyassarov/nri-plugins/actions/runs/12763538771/workflow#L52.